python/examples/rpm-e.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
+
+# Uninstall a package from the system.
+# A Python equivalent for `rpm -e hello`
+
+import sys
+import rpm
+
+ts = rpm.TransactionSet()
+for name in sys.argv[1:]:
+    ts.addErase(name)
+ts.run(lambda *args: ..., "")
```

Comment:
A transaction with nothing but erase elements probably does happen to work. But one should always supply a callback to the transaction, especially these being "official" examples. 

Maybe demonstrate some other callback events for this? Or combine with the install example, because we don't want people to think one can only do one kind of operation in a transaction even if the rpm cli limits you that way.

Reply:
> But one should always supply a callback to the transaction, especially these being "official" examples.

I agree. And even though the lambda works there, it's quite WTF-looking.
Updated, PTAL

> Maybe demonstrate some other callback events for this? Or combine with the install example, because we don't want people to think one can only do one kind of operation in a transaction even if the rpm cli limits you that way.

This is what I don't want to do. My plan was to explain this in the documentation page, that it is possible to combine different operations within one transaction. If you want to, we can add some python example showing it.

But I would like to keep this example as simple as possible, showing just how to do `rpm -e hello`

Reply:
I just think it gives you a rather strange impression of the API when it's demonstrated as two separate cases with two entirely different callbacks. But okay, that can be resolved with two simple comments, something like below, respectively for erase and install examples:

```
# erasure-only transactions don't strictly require anything from the callback
# most operations require at least rpm.RPMCALLBACK_INST_OPEN_FILE  and rpm.RPMCALLBACK_INST_CLOSE_FILE to be handled
```

Reply:
I put the comment into the `rpm-e.py` callback